QgisModelBaker/libqgsprojectgen/db_factory/db_simple_factory.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+/***************************************************************************
+    begin                :    08/04/19
+    git sha              :    :%H$
+    copyright            :    (C) 2019 by Yesid Polania
+    email                :    yesidpol.3@gmail.com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from .pg_factory import PgFactory
+from .gpkg_factory import GpkgFactory
+from QgisModelBaker.libili2db.globals import DbIliMode, displayDbIliMode
+
+
+class DbSimpleFactory:
```

Comment:
Done!
*Why is it a "simple" factory?*
Because the **create_factory** method provides a single point to instantiate the DbFactory Classes. This avoids duplicate *if/else* statements to instantiate DbFactory.